state_machine/applications/flight/lib/message.py

```diff
@@ -1,4 +1,12 @@
 class Message:
+    """The most basic message type. Supports ascii messages no longer than 251 bytes.
```

Comment:
I would also comment that all other messages should inherit from this one.

Reply:
Good idea